### ISSUE TITLE
CSVLink warning for `asyncOnClick`

### ISSUE THREAD

**Issue by standingwave** [abdennour/react-csv#104]
Recently started seeing this warning with usages of `CSVLink`.

```
Warning: React does not recognize the `asyncOnClick` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `asynconclick` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
          in a (created by CSVLink)
```

**Comment by zindzay** [comment]
> Recently started seeing this warning with usages of `CSVLink`.
> 
> ```
> Warning: React does not recognize the `asyncOnClick` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `asynconclick` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
>           in a (created by CSVLink)
> ```

 It is now fixed with version: 1.0.15-RC1

**Comment by TranDucBlog** [comment]
I upgraded to version 1.0.15 but still warned
`Warning: React does not recognize the `asyncOnClick` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `asynconclick` instead. If you accidentally passed it from a parent component, remove it from the DOM element.`

**Comment by zindzay** [comment]
I have the same problem, version 1.0.15-RC1

**Comment by yulya9060** [comment]
I have the same problem, version 1.0.15-RC1.
Warning: React does not recognize the `asyncOnClick` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `asynconclick` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

**Comment by abdennour** [comment]
If someone is interested in being a contributor in this project, please let [contact me](http://in.abdennoor.com).

While it's hard to review all issues/PR due to time limitation, it is also not acceptable to approve PRs without review.

**Comment by TranDucBlog** [comment]
This error has been fixed in version 1.0.15-RC2

**Comment by mccabemj** [comment]
Closed due to 1.0.15-RC2 fix
